detox/src/devices/Device.js

```diff
@@ -86,6 +86,11 @@ class Device {
 
     await this.deviceDriver.waitUntilReady();
     await this.deviceDriver.waitForActive();
+    await this.deviceDriver.emitter.emit('appReady', {
```

Comment:
Cannot put emitting inside that `waitForActive`, because it is used elsewhere (e.g., matchFace, unmatchFace).

I am hesitant to put that in `waitUntilReady` because I suspect there might be a chance something goes awry with `waitForActive`, so I want to emit as late as possible, to not miss a problem with the app.

Reply:
It appears `waitForActive()` is scarce on Android. @rotemmiz why this difference between platforms?